### DISCUSSION TITLE
any way to support hv on M2 iPad iPadOS 16.6

### DISCUSSION THREAD

**Discussion by HH7H** [utmapp/UTM#6026] [UTM Help (iOS)]
??

**Comment by tangtao-xp1** [comment]
Apple removed Hypervisor support from XNU in iOS 16.4. Here is a diff of iOS 16.3.1 and iOS 16.4. What this means is that even if a jailbreak/TrollStore comes out for iOS 16.6.1/17.0, there will not be UTM virtualization support, even on M1/M2 iPads.

https://x.com/utmapp/status/1708907045314035986
